src/test/groovy/org/prebid/server/functional/tests/TargetingSpec.groovy

```diff
@@ -418,19 +420,19 @@ class TargetingSpec extends BaseSpec {
         assert response.targeting.isEmpty()
     }
 
-    def "PBS amp should make right calculation for hb_pb targeting"() {
+    def "PBS amp should use ranges.max value for hb_pb targeting when bid.price biggest that ranges.max"() {
```

Comment:
maybe `PBS amp should use ranges.max value for hb_pb targeting when bid.price is greater than ranges.max`?